pkg/tfbridge/config_encoding.go

```diff
@@ -164,12 +134,22 @@ func (enc *configEncoding) UnmarshalPropertyValue(key resource.PropertyKey, v *s
 		return &v, nil
 	}
 
+	// Computed sentinels are coming in as always having an empty string, but the encoding coerses them to a zero
+	// value of the appropriate type.
+	if pv.IsComputed() && gotShimType {
```

Comment:
Probably being overcautious here; current prod code does not peek inside a Computed and if it's an empty string inside it's ok; still having a type-appropriate zero value seems more future-proof.